### PR TITLE
Calling companyProfile service only once.

### PR DESCRIPTION
**Pull request by sampankumar**
Calling companyProfile service only once.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/controllers/features/limited/companyLookupController.ts [hunk 1]
```diff
@@ -43,20 +42,46 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
...
         next(error);
-
     }
 };
 
 const getPageProperties = (errors?: FormattedValidationErrors) => ({
     errors
 });
+
+async function getCompanyDetails (companyLookupService: CompanyLookupService, session: Session, companyNumber: string, req: Request) {
```
Comment (swalke-ch) [inline src/main/controllers/features/limited/companyLookupController.ts:73]: checking if the new function is covered with unit tests?

**Review by swalke-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sardhani-ch — APPROVED** [review]

**Review by atedros-ch — APPROVED** [review]

**Pull request merged**